app/modules/import-export-customization/assets/js/admin.js

```diff
@@ -0,0 +1,70 @@
+import { AppsEventTracking } from '../../../../assets/js/event-track/apps-event-tracking';
+
+class Admin {
+	constructor() {
+		const urlParams = new URLSearchParams( window.location.search );
+
+		if ( 'elementor-tools' === urlParams.get( 'page' ) ) {
+			this.sendPageToolsViewedEvent();
+
+			navigation.addEventListener( 'navigate', ( event ) => {
```

Comment:
I don't think we can use the Navigation API yet: https://developer.mozilla.org/en-US/docs/Web/API/Navigation_API